test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -28,6 +28,54 @@ public PackCommandTests(MsbuildIntegrationTestFixture fixture)
             msbuildFixture = fixture;
         }
 
+        [PlatformFact(Platform.Windows)]
+        public void CreatePackageWithFiles()
```

Comment:
The name doesn't really tell me anything here. 

Consider `MethodName_StateUnderTest_ExpectedBehavior`